packages/frontend/src/components/Claim/WinBid/WinBidForm.tsx

```diff
@@ -41,6 +41,11 @@ export const WinBidForm = ({ userBid, withdrawalAmount, setView, voucher, setVou
       {!userBid.claimed && userBid.winType !== WinType.Auction && (
         <WinOption>
           <span>{withdrawText[userBid.winType]}</span>
+          {isWinningBid && !userBid.claimed && (
```

Comment:
Participants that haven't won in raffle/auction should see deadline as well, because they can withdraw bid funds. Also `!userBid.claimed` can be dropped as component is only rendered when user hasn't yet claimed funds.